spec/models/post_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  before(:all) do
+    @user = User.create(
+      name: 'John Doe',
+      photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+      bio: 'Teacher from Mexico.',
+      posts_counter: 0
+    )
+
+    @post = Post.create(
+      title: 'Hello World!',
+      text: 'This is my first post.',
+      comments_counter: 0,
+      likes_counter: 0,
+      user_id: @user.id
+    )
+  end
+  it '@post should be valid' do
+    expect(@post).to be_valid
+  end
+  it '@post should have 250 maximum length' do
+    @post.title = 'Helo World!'
+    expect(@post).to be_valid
+  end
+  it '@post comments_counter to be an integer greater than or equal to 0' do
+    @post.comments_counter = 'one'
+    expect(@post).to_not be_valid
+  end
+  it '@post likes_counter to be an integer greater than or equal to 0' do
+    @post.likes_counter = 'one'
+    expect(@post).to_not be_valid
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md#:~:text=Add%20unit%20specs%20for%20all%20of%20your%20models%27%20methods%20and%20validations.), Please add a unit test for the `most_recent_comments` method.